src/sprite_lists.py

```diff
@@ -0,0 +1,27 @@
+import arcade
+
+
+
+class SpriteLists:
+    """
+    Structure that contains SpriteLists
+    """
+
+    player_sprite_list: arcade.SpriteList
+    projectile_sprite_list: arcade.SpriteList
+    beam_sprite_list: arcade.SpriteList
+    wall_sprite_list: arcade.SpriteList
+
+    def __init__(self):
+        self.player_sprite_list = arcade.SpriteList()
+        self.projectile_sprite_list = arcade.SpriteList()
+        self.beam_sprite_list = arcade.SpriteList()
+        self.wall_sprite_list = arcade.SpriteList()
```

Comment:
Not a blocker: I'm cool with renaming these to `self.players`, `self.projectiles`, etc.  Since the class is called `SpriteLists` it still feels obvious what they are.

Reply:
I see we're creating a `wall_sprite_list` here, but then replacing it with a new list from `Arena`.  Relates to my other comment about passing `SpriteLists` into the arena loading code, to match the pattern we use everywhere else.

Reply:
So for example, other functions would call `sprite_lists.players`. I like that a lot actually. I'll implement.